core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -207,6 +207,7 @@ private int formatFiles(ArgInfo argInfo, JavaFormatterOptions options) {
     int numThreads = Math.min(MAX_THREADS, argInfo.parameters.fileNamesFlag.size());
     ExecutorService executorService = Executors.newFixedThreadPool(numThreads);
 
+    Map<Path, String> inputs = new LinkedHashMap<>();
```

Comment:
Sure, didn't want to obscure the logic behind the simple 3-line implementation.

What would the result look like? Return `FormatFileCallable::this` and the caller cares to extract the information needed - or a dedicated `FormatResult` class that only contains the needed information like `String formatted` and `boolean changed`. What else?